HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -3097,7 +3102,7 @@ def self.apply_hvac_sizing(runner, hpxml_bldg, weather, cfa)
                 [HPXML::HVACTypeBoiler,
                  HPXML::HVACTypeElectricResistance].include?(htg_sys.heating_system_type))
           htg_sys.heating_airflow_cfm = Float(hvac_sizing_values.Heat_Airflow.round)
-          htg_sys.heating_airflow_cfm_isdefaulted = true
+          htg_sys.heating_airflow_cfm_isdefaulted = hvac_sizing_values.Heat_Airflow_isdefaulted
```

Comment:
Now, this may or may not be defaulted. It's defaulted when not provided in the HPXML file.